src/network/webtask.cpp

```diff
@@ -236,17 +236,9 @@ void WebTask::slotStart(int timeoutMillis) {
             Qt::UniqueConnection);
 }
 
-void WebTask::makeStatePending() {
-    m_state = State::Pending;
-}
-
-void WebTask::makeStateFinished() {
-    m_state = State::Finished;
-}
-
 void WebTask::slotAbort() {
```

Comment:
I think you also need to implement doNetworkReplyAborted and kill our Qtimer

Reply:
Is there a way for me to see if the QTimer is still running in any case?

May I see it with the debugging mode that you've showed me?